test/unit/search/search_configuration_test.cpp

```diff
@@ -52,6 +52,114 @@ TYPED_TEST(search_configuration_test, configuration_exists)
     EXPECT_TRUE(decltype(cfg)::template exists<TypeParam>());
 }
 
+TEST(search_configuration_test, max_error_defaults)
+{
+    // empty config defaults to 0 for every error
+    EXPECT_EQ((seqan3::search_cfg::max_error{}.value),
```

Comment:
E capitalized? (If so, then also for all following comments) Plus a dot in the end?
If not capitalized, than I would write the ifs also with an uncapitalized i, to be consistent.

Reply:
I made the other comments start with a lower case i since they are not proper sentences (so no dot)